<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>413</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kenwood Database Generator</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../appresources.qrc">
    <normaloff>:/icons/applicationIcon</normaloff>:/icons/applicationIcon</iconset>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="handleWidth">
       <number>3</number>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="driveListContainer" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="driveListLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="driveListLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>&amp;USB Drives</string>
          </property>
          <property name="buddy">
           <cstring>driveList</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListView" name="driveList">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="terminalLogContainer" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>3</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>0</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="terminalListLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="terminalLogLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>3</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>&amp;Terminal Output</string>
          </property>
          <property name="buddy">
           <cstring>terminalLogWindow</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="terminalLogWindow">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>3</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>250</width>
            <height>0</height>
           </size>
          </property>
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <property name="documentTitle">
           <string/>
          </property>
          <property name="undoRedoEnabled">
           <bool>false</bool>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="plainText">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="contextMenuPolicy">
    <enum>Qt::PreventContextMenu</enum>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionRefresh"/>
   <addaction name="actionGenerate"/>
   <addaction name="actionView"/>
   <addaction name="separator"/>
   <addaction name="actionAbout"/>
  </widget>
  <action name="actionRefresh">
   <property name="icon">
    <iconset resource="../appresources.qrc">
     <normaloff>:/actions/refresh</normaloff>:/actions/refresh</iconset>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
   <property name="toolTip">
    <string>Refresh Drive List</string>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
  </action>
  <action name="actionGenerate">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../appresources.qrc">
     <normaloff>:/actions/write</normaloff>:/actions/write</iconset>
   </property>
   <property name="text">
    <string>Generate</string>
   </property>
   <property name="toolTip">
    <string>Write Database to Drive</string>
   </property>
  </action>
  <action name="actionView">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../appresources.qrc">
     <normaloff>:/actions/open</normaloff>:/actions/open</iconset>
   </property>
   <property name="text">
    <string>View</string>
   </property>
   <property name="toolTip">
    <string>Show Database on Drive</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="../appresources.qrc">
     <normaloff>:/actions/about</normaloff>:/actions/about</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionClearLog">
   <property name="icon">
    <iconset resource="../appresources.qrc">
     <normaloff>:/actions/clear</normaloff>:/actions/clear</iconset>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../appresources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionRefresh</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>refreshActionTriggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>301</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionView</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>viewActionTriggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>301</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>terminalLogWindow</sender>
   <signal>customContextMenuRequested(QPoint)</signal>
   <receiver>MainWindow</receiver>
   <slot>terminalLogCustomContextMenuRequested(QPoint)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>376</x>
     <y>237</y>
    </hint>
    <hint type="destinationlabel">
     <x>301</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionClearLog</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>clearLogActionTriggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>301</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionGenerate</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>generateActionTriggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>301</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>refreshActionTriggered()</slot>
  <slot>viewActionTriggered()</slot>
  <slot>terminalLogCustomContextMenuRequested(QPoint)</slot>
  <slot>clearLogActionTriggered()</slot>
  <slot>generateActionTriggered()</slot>
 </slots>
</ui>
